README.md

```diff
@@ -56,6 +56,22 @@ sudo apt install libsdformat<#>-dev libsdformat<#>
 Be sure to replace `<#>` with a number value, such as 2 or 3, depending on
 which version you need, or leave it empty for version 1.
 
+### macOS
+
+On macOS, add OSRF packages:
+  ```sh
+  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
this install command is out of date (see https://brew.sh for the latest)

~~~
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
~~~